frontend/public/metalkube/components/host/host-detail.tsx

```diff
@@ -34,6 +40,8 @@ const BaremetalHostDetails = ({ host }) => {
     'fa-refresh': true,
   });
 
+  const totalCapacity = getHostTotalStorageCapacity(host) / 1024;
```

Comment:
Nitpick: what about using conversion `units.humanize()` from `frontend/public/components/utils/units.js`?
Will work for different units as well.